app/components/candidate_interface/application_form_course_choices_component.html.erb

```diff
@@ -1,7 +1,7 @@
 <section class="govuk-!-margin-bottom-8">
   <h2 class="govuk-heading-m govuk-!-font-size-27"><%= t('section_groups.courses') %></h2>
 
-  <% unless choices_are_present? %>
+  <% unless number_of_choices == 3 %>
```

Comment:
Is it possible to move this method to the component? And is it possible to use the constant from the application form instead of hardcoding the number 3 here in the view or component?

Reply:
Both done ♻️ 😄